Client/src/Components/DataCell/DataCell.js

```diff
@@ -17,6 +23,17 @@ export default function DataCell() {
         e.preventDefault();
 
         const json = e.target.jsonFile.files[0];
+        try {
+            //so if this here is not proper json extension it will catch
+            var ext = getExtension(json.name);
+            if (ext != '.json') {
+                throw "File extension is not supported";
```

Comment:
Does this work? I dont know if you can just throw as such without indicating the type of error

Reply:
this works, but now i have changed it console.error('file extension is not supported') to avoid confusion